### ISSUE TITLE
[Bug]: Large error when trying to build ith

### ISSUE THREAD

**Issue by dovahkiin98** [Baseflow/flutter-geolocator#1549]
### Please check the following before submitting a new issue.

- [X] I have searched the [existing issues](https://github.com/baseflow/flutter-geolocator/issues).
- [X] I have carefully [read the documentation](https://github.com/Baseflow/flutter-geolocator/blob/main/geolocator/README.md) and verified I have added the required platform specific configuration.

### Please select affected platform(s)

- [X] Android
- [ ] iOS
- [ ] Linux
- [ ] macOS
- [ ] Web
- [ ] Windows

### Steps to reproduce

1. Use version 13.0.0
2. Build application

### Expected results

Application runs normally

### Actual results

Very large error, mostly JS stuff `Error: Type 'JSAny' not found.`

### Code sample

geolocator: 12.0.0

### Version

13.0.0

### Flutter Doctor output

<details><summary>Doctor output</summary>

```console
[√] Flutter (Channel stable, 3.22.3, on Microsoft Windows [Version 10.0.22635.3790], locale en-GB)
    • Flutter version 3.22.3 on channel stable at J:\Programming\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision b0850beeb2 (3 weeks ago), 2024-07-16 21:43:41 -0700
    • Engine revision 235db911ba
    • Dart version 3.4.4
    • DevTools version 2.34.3

[√] Windows Version (Installed version of Windows is version 10 or higher)

[√] Android toolchain - develop for Android devices (Android SDK version 35.0.0)
    • Android SDK at J:\Programming\Android\Android_SDK
    • Platform android-35, build-tools 35.0.0
    • ANDROID_HOME = J:\Programming\Android\Android_SDK
    • ANDROID_SDK_ROOT = J:\Programming\Android\Android_SDK
    • Java binary at: J:\Programming\Android\Android_Studio\jbr\bin\java
    • Java version OpenJDK Runtime Environment (build 17.0.11+0--11852314)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop Windows apps (Visual Studio Community 2022 17.4.4)
    • Visual Studio at J:\VisualStudio\Community
    • Visual Studio Community 2022 version 17.4.33213.308
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 2024.1)
    • Android Studio at J:\Programming\Android\Android_Studio_Canary
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 21.0.3+-12099254-b509.4)

[!] Android Studio (version unknown)
    • Android Studio at J:\Programming\Android\Android_Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    X Unable to determine Android Studio version.
    • android-studio-dir = J:\Programming\Android\Android_Studio
    • Java version OpenJDK Runtime Environment (build 17.0.11+0--11852314)

[√] IntelliJ IDEA Community Edition (version 2024.1)
    • IntelliJ at J:\IntelliJ IDEA
    • Flutter plugin version 80.0.2
    • Dart plugin version 241.18808

[√] VS Code (version 1.91.1)
    • VS Code at C:\Users\ahmad\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.52.0

[√] Connected device (4 available)
    • sdk gphone64 x86 64 (mobile) • emulator-5554 • android-x64    • Android 14 (API 34) (emulator)
    • Windows (desktop)            • windows       • windows-x64    • Microsoft Windows [Version 10.0.22635.3790]
    • Chrome (web)                 • chrome        • web-javascript • Google Chrome 126.0.6478.128
    • Edge (web)                   • edge          • web-javascript • Microsoft Edge 126.0.2592.24

[√] Network resources
    • All expected network resources are available.

! Doctor found issues in 1 category.
```

</details>

**Comment by AlexR02** [comment]
I'm experiencing the same errors in the 13.0.0 version.

**Comment by devlprRohit** [comment]
i am also experiencing the error

**Comment by mvanbeusekom** [comment]
Hi @dovahkiin98, @AlexR02 and @devlprRohit,

Apologies for the inconvenience, by accident a reference to the dart:js_interop was referenced through to the non-web platforms causing the error.

I have just released version 13.0.1 resolving these issues.
